Homework/Nam_Nguyen/Lesson11/LeetCode_259_3SumSmaller.java

```diff
@@ -0,0 +1,38 @@
+package Homework.Nam_Nguyen.Lesson11;
+
+import java.util.Arrays;
+
+public class LeetCode_259_3SumSmaller {
+
+    public int threeSumSmaller(int[] nums, int target) {
+        int res = 0;
+        int length = nums.length;
+        Arrays.sort(nums);
+        
+        for (int i = 0; i < length - 1; i++) {
+            int left = i + 1, right = length - 1;
+            int k = target - nums[i];
+            while(left < right) {
+                if(nums[left] + nums[right] < k) {
+                    int pointer = right;
+                    while(left < pointer) {
```

Comment:
i think can improve this part: res = res + (right-left+1) ?

Reply:
@quangh33 Yes. You're right. I improved that

Reply:
ok please push the updated code